extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.ViewPageTarget.init.js

```diff
@@ -228,9 +227,8 @@
 			ve.track( 'mwedit.init', { type: 'page', mechanism: 'click' } );
 
 			if ( history.pushState && uri.query.veaction !== 'edit' ) {
-				if (!pageExists) {
-					veEditUri.extend( { flow: 'create-page-create-button' } );
-				}
+				mw.hook('ve.afterVEInit').fire(veEditUri);
+
 				// Replace the current state with one that is tagged as ours, to prevent the
 				// back button from breaking when used to exit VE. FIXME: there should be a better
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1134)